packages/charts/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -693,25 +700,40 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     const { xBarScale, yBarScale } = this._getScales(containerHeight, containerWidth);
     const colorScale = this._createColors();
 
+    const baselinePoint = this._yMax < 0 ? this._yMax : 0;
```

Comment:
>this._yMax [](http://example.com/codeflow?start=26&length=10)

This should be < 0 only in case this.props.yMaxValue < 0 and yMax < this.props.yMaxValue

Reply:
this._ymax is calculated as max(all y points, this.props.yMaxValue). So won't this._ymax < 0 be sufficient condition?

Reply:
sounds fine.